### PR TITLE
var idea

### PR DESCRIPTION
**Pull request by archae0pteryx**
just an idea... thought i'd throw it out there... love what you've done here! the simplicity is great!
Not sure how much this adds but it would be nice to have some easy overrides... like i said. just a thought.

### CODE REVIEW

**Comment by Ajusa** [comment]
Thank you very much!
I definitely agree about having easy overrides, right now the document recommends trying to do a "ctrl-f" and replacing the values you want. I also didn't realize that most modern browsers support CSS next. 

The only problem I have with your pull request is that it does add more bytes than I would like. Thankfully, there is a "PostCSS" [plugin](https://github.com/MadLittleMods/postcss-css-variables) that can transform what you wrote directly to what I have, by replacing each variable with the correct value. 

I will probably merge your PR in once I have this PostCSS stuff figured out.

File: lit.css [hunk 1]
```diff
@@ -4,7 +4,16 @@
...
 */
-
+:root {
+    --primary-font: Nunito;
+    --primary-color: #212121;
+    --secondary-color: #dbdbdb;
+    --accent-color: #234567;
+    --link-color: gold;
+    --border-color: green;
+    --primary-button-bg: gold;
```
Comment (Ajusa) [inline lit.css]: Hey, could you change the "gold" back to the original hexadecimal value of #FA0? #FA0 saves one extra byte with gzip for some reason. If you want, you could add a comment after the "#FA0" saying "gold".

Other than that, I think that this is a great idea and will merge as soon as I figure out the PostCSS stuff!

**Comment by Ajusa** [comment]
Just finished getting all Gulp and PostCSS set up for the first time. I'll try and do a merge here

**Comment by Ajusa** [comment]
I did merge this in, but then I changed some significant parts of it. I liked the idea, so I kept most of the variables, and I renamed a few to make more sense to me. 

I don't plan on adding secondary buttons or more components anytime soon.

**Comment by archae0pteryx** [comment]
awesome! i’m glad you liked the idea! doing a find to replace seems like it might be a problem later if you ever update the repo. this way someone could define their own ‘primitive’ vars 

that’s sweet! keep up the great work!

**Comment by Ajusa** [comment]
Yeah, good point. Hope you enjoy using it!

**Pull request merged**